src/Symfony/Component/Console/Cursor.php

```diff
@@ -28,82 +27,114 @@ public function __construct(OutputInterface $output, $input = STDIN)
         $this->input = $input;
     }
 
-    public function moveUp(int $lines = 1)
+    public function moveUp(int $lines = 1): self
```

Comment:
since the class is not final (should it be?), we need to add `@return $this` anotations

Reply:
Class is now final :)